### PR TITLE
Create a section for the Inspector + Micro

### PR DESCRIPTION
**Pull request by stanch**

### CODE REVIEW

File: docs/testing-debugging/snowplow-inspector/adding-schemas/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: "Adding custom schemas"
+sidebar_position: 2
+description: "How to add custom schemas to Snowplow Inspector"
+---
+
+By default, the extension is able to see Iglu schemas for Events and Entities that are generated by the official Snowplow [trackers](/docs/collecting-data/collecting-from-own-applications/index.md), as they are publicly available on the public [Iglu Central](/docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-central/index.md) registry.
```
Comment (stanch) [inline docs/testing-debugging/snowplow-inspector/adding-schemas/index.md:7]: Copied verbatim (aside from amending the image paths and links)

File: docs/testing-debugging/snowplow-inspector/importing-events/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Importing events"
+sidebar_position: 2
+description: "How to import events from other devices"
+---
+
+In the main Event Debugger view, the Event Timeline down the side allows you to import events from other devices that you can view as if the were generated by your own browser.
```
Comment (stanch) [inline docs/testing-debugging/snowplow-inspector/importing-events/index.md:7]: Copied verbatim (aside from amending the image paths and links)

File: static/_redirects [hunk 3]
```diff
@@ -156,4 +156,7 @@ docs/understanding-tracking-design/managing-data-structures-with-data-structures
 
 /docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/* /docs/getting-started-on-snowplow-open-source/quick-start-aws/ 301
 /docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/* /docs/getting-started-on-snowplow-open-source/quick-start-gcp/ 301
-/docs/getting-started-on-snowplow-open-source/what-is-quick-start/faq/ /docs/getting-started-on-snowplow-open-source/faq/ 301
\ No newline at end of file
+/docs/getting-started-on-snowplow-open-source/what-is-quick-start/faq/ /docs/getting-started-on-snowplow-open-source/faq/ 301
+
+/docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/* /docs/testing-debugging/snowplow-inspector/:splat 301
+/docs/getting-started-with-micro/* /docs/testing-debugging/snowplow-micro/:splat 301
```
Comment (stanch) [inline static/_redirects:162]: This should work for the parent page as Nelify will not redirect for pages that exist

File: docs/getting-started-with-micro/index.md [hunk 4]
```diff
@@ -3,8 +3,6 @@ title: "Using Snowplow Micro locally"
 sidebar_position: 25
 ---
 
-```mdx-code-block
-import DocCardList from '@theme/DocCardList';
+[Snowplow Micro](/docs/testing-debugging/snowplow-micro/what-is-micro/index.md) is a minified version of the Snowplow pipeline that can run on your machine. For a few quick tests, Micro can be a simple alternative to deploying an full-blown Snowplow Open Source pipeline to your cloud. However, note that Micro does not persist the data in any warehouse or database — you can just inspect it using the API while Micro is running.
```
Comment (jethron) [inline docs/getting-started-with-micro/index.md:6]: Nit: I think "you can only inspect it using the API while Micro is running" reads slightly better than "just" in this case.

File: docs/testing-debugging/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Testing and debugging"
+sidebar_position: 57
```
Comment (jethron) [inline docs/testing-debugging/index.md:3]: I feel like ~65 is a more natural position?

`Defining > Collecting > Testing > Enriching` vs `Testing > Defining > Collecting > Enriching`?

**Review by jethron — APPROVED** [review]
LGTM, thanks for splitting up the extension stuff!

File: docs/testing-debugging/index.md (hunk 5 above)
Comment (stanch) [reply docs/testing-debugging/index.md]: I actually did consider this, can’t remember why I went with 57. Changed, thanks for the reminder!

**Pull request merged**